### PR TITLE
105 release notes

### PR DESCRIPTION
**Pull request by Rumyra**
### Description

Updating firefox 105 release notes for release this week

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/105/index.md [hunk 1]
```diff
@@ -10,48 +10,29 @@ tags:
 
 {{FirefoxSidebar}}
 
-This article provides information about the changes in Firefox 105 that will affect developers. Firefox 105 is the current [Beta version of Firefox](https://www.mozilla.org/en-US/firefox/channel/desktop/#beta) and will ship on [September 20, 2022](https://wiki.mozilla.org/RapidRelease/Calendar#Future_branch_dates).
+This article provides information about the changes in Firefox 105 that will affect developers. Firefox 105 was released on September 20, 2022.
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/105/index.md:13]: I guess we can remove future tense :)

```suggestion
This article provides information about the changes in Firefox 105 that affect developers. Firefox 105 was released on September 20, 2022.
```

File: files/en-us/mozilla/firefox/releases/105/index.md [hunk 2]
```diff
@@ -10,48 +10,29 @@ tags:
...
-#### Removals
-
-### Security
-
-#### Removals
+No notable changes.
 
 ### APIs
 
 #### DOM
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/105/index.md:32]: (unrelated) it's not clear that "DOM" is a subsection of "APIs" in the generated HTML:

image

**Review by dipikabh — APPROVED** [review]
Thanks a lot @Rumyra! LGTM.

**Review by bsmth — APPROVED** [review]
Looks good, one minor wording suggestion open 👍🏻

File: files/en-us/mozilla/firefox/releases/105/index.md (hunk 2 above)
Comment (Rumyra) [reply files/en-us/mozilla/firefox/releases/105/index.md:32]: That's a styling issue with h3/h4, maybe something to bring up on yari?

File: files/en-us/mozilla/firefox/releases/105/index.md [hunk 3]
```diff
@@ -10,48 +10,29 @@ tags:
...
 
-#### Media, WebRTC, and Web Audio
-
-#### Removals
-
-### WebAssembly
-
-#### Removals
 
 ### WebDriver conformance (WebDriver BiDi, Marionette)
```
Comment (Rumyra) [inline files/en-us/mozilla/firefox/releases/105/index.md:37]:
```suggestion
### WebDriver conformance (WebDriver BiDi, Marionette)
```

File: files/en-us/mozilla/firefox/releases/105/index.md [hunk 4]
```diff
@@ -69,9 +50,6 @@ This article provides information about the changes in Firefox 105 that will aff
 
 - Support for defining persistent scripts using {{WebExtAPIRef("scripting")}} has been added. A script is identified as persistent using the `persistAcrossSessions` property in {{WebExtAPIRef("scripting.RegisteredContentScript")}} ({{bug("1751436")}}).
 
-### Removals
-
-### Other
 
 ## Older versions
```
Comment (Rumyra) [inline files/en-us/mozilla/firefox/releases/105/index.md:54]:
```suggestion
## Older versions
```

File: files/en-us/mozilla/firefox/releases/105/index.md (hunk 2 above)
Comment (bsmth) [reply files/en-us/mozilla/firefox/releases/105/index.md:32]: Opened an issue in https://github.com/mdn/yari/issues/7204

**Pull request merged**